packages/checkout-storefront/src/sections/ShippingAddressSection/ShippingAddressSection.tsx

```diff
@@ -38,20 +41,18 @@ export const ShippingAddressSection: React.FC<CommonSectionProps> = ({ collapsed
 
   const updateShippingAddress = useCallback(
     async ({ autoSave, ...address }: AddressFormData) => {
-      const result = await checkoutShippingAddressUpdate({
-        checkoutId: checkout.id,
-        shippingAddress: getAddressInputData(address),
-        validationRules: getAddressVlidationRulesVariables(autoSave),
-      });
-
-      const [hasErrors, errors] = extractMutationErrors(result);
-
-      if (hasErrors) {
-        showErrors(errors, "checkoutShippingUpdate");
-        setApiErrors(errors);
-      }
+      //   const result = await checkoutShippingAddressUpdate({
```

Comment:
Should this be removed?

Reply:
Aaah yeah, commented for debug, uncommenting